pkg/signature/manifest.go

```diff
@@ -133,20 +134,9 @@ func GetDigest(manifest Manifest, domain Domain) (digest [32]byte, yml []byte, e
 	return digest, yml, err
 }
 
+//See ./testdata/sign_file.sh for a shell script implementation.
+//This here differs in that it normalizes quoting while the shell script doesnt (yet).
 func ValidateSignature(manifest Manifest, domain Domain) (valid bool, digest [32]byte, yml []byte, err error) {
-	// Shell equivalent
```

Comment:
We should not keep this comment up-to-date, the shell script is a better reference.